lib/validate_doc_update.js

```diff
@@ -24,7 +24,7 @@ var isDbAdmin = function(userCtx, secObj) {
 var allowed = function(newDoc, oldDoc, userCtx, secObj) {
 
     // ensure logged in
-    if (userCtx.name === null) {
+    if (typeof userCtx.name === 'undefined' || userCtx.name === null) {
```

Comment:
Well that could potentially be less backward compatible. I have no idea why a null test is there!